docs/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md

```diff
@@ -0,0 +1,14 @@
+---
+id: platforms
+title: "Camunda Platform 8 officially supported Kubernetes platforms"
```

Comment:
I'd ideally like to see a shorter title -- perhaps "Supported Kubernetes platforms"?